logs.go

```diff
@@ -91,6 +92,23 @@ func getLoggableResources(getPodContainers func(string, string) ([]string, error
 	return loggableResources, nil
 }
 
+// GetProjectPodsBySubstr returns a list of pod names that contain the provided string, in the named project.
+func getProjectPodNamesBySubstr(project, substr string) ([]string, error) {
```

Comment:
Wouldn't it be better if instead of this function we would try to find a `*nagios*` pod in all `projects` and print an error (in main.go) in case none was found?